scripts/unbonding_batches_visualisation/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "unbonding_batches_visualisation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "raw": "node -r ts-node/register --env-file=./.env src/app.ts",
+    "start": "node -r ts-node/register --env-file=./.env src/app.ts | jq -r '([\"id\",\"status\",\"expected\",\"creation(UTC)\",\"finalization(UTC)\",\"unstaked\"] | (., map(length*\"-\"))), (.[] | [.batch_id,.status,.expected_amount // \"-\",.creation_time,.expected_finalization_time // \"-\",.unstaked_amount // \"-\"]) | @tsv' --raw-output | column -t"
```

Comment:
interesting solution, but maybe you can add this past processing into typescript itself? `jq` is ok, but hard to read in this case, and requires additional software to be installed on the machine. I think it is better to implement same logic but on typescript.